contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+└── wg-resource-management
+```
+
+where each SIG or working group is given a top level directory with components
+maintained by the SIG listed in sub directories. For newcomers to the community
+an `unsorted-to-be-used-by-newcomers-only` directory may be used before an RFC
+can be properly routed to a SIG although hopefully if discussion for a potential
+RFC begins on the mailing lists proper routing information will be provided to
+the RFC author. Additionally a top level index of RFCs may be helpful for people
+looking for a complete list of RFCs. There should be basic CI to ensure that a
+`index.md` remains up to date.
+
+Ideally no work would begin on an RFC before it has been approved by the
+responsible SIG or working group. In order to help combat the explosion of GitHub
+issues against `kubernetes/kubernetes` it is further proposed for each chartered
+SIG to create a top level repository within the `kubernetes` GitHub organization
+where implementation issues would be filed and linked to in the RFC.
+
+### RFC Editor Role
+
+Taking a cue from the [Python PEP process][], I believe that a group of RFC editors
+will be required to make this process successful; the job of an RFC editor is
+likely very similar to the [PEP editor responsibilities][] and will hopefully
+provide another opportunity for people who do not write code daily to contribute
+to Kubernetes.
+
+[Python PEP process]: https://www.python.org/dev/peps/pep-0001/
+[PEP editor responsibilities]: https://www.python.org/dev/peps/pep-0001/#pep-editor-responsibilities-workflow
+
+### Important Metrics
+
+I believe the primary metrics which would signal the success or failure of the RFC
+process would be
+
+- distribution of RFC merge times
+- RFC rejection rate
+- distribution of time between RFC acceptance and implementation issue creation
+- PRs referencing an RFC merged per week
+
+### Customization of the RFC Process
+
+Given the general independence of SIGs I believe it will be important for SIGs to
+decide if they would like to opt out of the RFC process entirely or if they would
+like to modify the RFC process to better serve their needs. For example I can
+imagine RFCs targeting SIG Architecture or SIG API Machinery would likely require
+a section on backwards compatibility and migration strategies. I believe the best
+place for a SIG to describe its development process would be in its charter.
+
+### Prior Art
+
+The RFC process as proposed was essentially stolen from the [Rust RFC process] which
+itself seems to be very similar to the [Python PEP process][]
+
+[Rust RFC process]: https://github.com/rust-lang/rfcs
+
+## Graduation Criteria
+
+Before we can consider this process successful for our current size I believe we
+should hit at least the following milestones
+
+- a release note draft can be generated by referring to RFC content
+- a road map can be generated by referring to RFC content
+
+The process for creating draft release notes and development status reports
```

Comment:
I disagree with the tie into road map and release notes.  I detailed my concerns above.